### PR TITLE
BDCat Staging release 1605301746

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from preprod.gen3.biodatacatalyst.nhlbi.nih.gov to staging.gen3.biodatacatalyst.nhlbi.nih.gov

### CODE REVIEW

**Review by Avantol13 — APPROVED** [review]

**Review by Avantol13 — CHANGES_REQUESTED** [review]
wait

File: staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json [hunk 1]
```diff
@@ -513,12 +520,12 @@
       "accessibleValidationField": "project_id"
     },
     "getAccessButtonLink": "https://dbgap.ncbi.nlm.nih.gov/",
-    "terraExportURL": "https://dev.terra.biodatacatalyst.nhlbi.nih.gov/#import-data",
```
Comment (Avantol13) [inline staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json:516]: we don't want to change this

File: staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json [hunk 2]
```diff
@@ -513,12 +520,12 @@
...
+    "terraExportURL": "https://terra.biodatacatalyst.nhlbi.nih.gov/#import-data",
     "terraTemplate": [
       "bdc",
       "gen3"
     ],
-    "sevenBridgesExportURL": "https://f4c-staging-vayu.sb.biodatacatalyst.nhlbi.nih.gov/import/windmill"
```
Comment (Avantol13) [inline staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json:521]: also don't want to change this

File: staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json [hunk 3]
```diff
@@ -612,20 +622,20 @@
         "rightIcon": "external-link"
       }
     ],
-    "terraExportURL": "https://dev.terra.biodatacatalyst.nhlbi.nih.gov/#import-data",
```
Comment (Avantol13) [inline staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json:615]: we don't want to change this

File: staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json [hunk 4]
```diff
@@ -612,20 +622,20 @@
...
+    "terraExportURL": "https://terra.biodatacatalyst.nhlbi.nih.gov/#import-data",
     "terraTemplate": [
       "bdc",
       "gen3"
     ],
-    "sevenBridgesExportURL": "https://f4c-staging-vayu.sb.biodatacatalyst.nhlbi.nih.gov/import/windmill",
```
Comment (Avantol13) [inline staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json:620]: we don't want to change this

File: staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json (hunk 3 above)
Comment (themarcelor) [reply staging.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json:615]: Let me fix this manually for now 👍

**Comment by themarcelor** [comment]
Closing this one to adopt a new PR:
https://github.com/uc-cdis/cdis-manifest/pull/2367

**Pull request closed without merging**